pkg/apis/proto/source/v1/udsource.proto

```diff
@@ -0,0 +1,138 @@
+syntax = "proto3";
```

Comment:
Is my understanding correct, all these definitions are only for underlining gRPC calls, we will not use them as the interfaces required for the users.

Reply:
Yes.